nextflow_schema.json

```
@@ -42,6 +43,322 @@
+                    "fa_icon": "far fa-file-code"
+                },
+                "probe_ref_name": {
+                    "type": "string",
+                    "default": "Tetrapods-UCE-5Kv1",
+                    "description": "Name of supported probe. Mandatory if not using `--probes` parameter.",
+                    "help_text": "Supported probes are 'Tetrapods-UCE-2.5Kv1', 'Tetrapods-UCE-5Kv1', 'Actinopterygians-0.5Kv1', 'Acanthomorphs-1Kv1', 'Arachnida-1.1Kv1', 'Coleoptera-1.1Kv1', 'Diptera-2.7Kv1', 'Hemiptera-2.7Kv1', 'Hymenoptera-1.5Kv1', 'Hymenoptera-2.5Kv2', and 'Anthozoa-1.7Kv1'",
+                    "enum": [
+                        "Tetrapods-UCE-2.5Kv1",
+                        "Tetrapods-UCE-5Kv1",
+                        "Actinopterygians-0.5Kv1",
+                        "Acanthomorphs-1Kv1",
+                        "Arachnida-1.1Kv1",
+                        "Coleoptera-1.1Kv1",
+                        "Diptera-2.7Kv1",
+                        "Hemiptera-2.7Kv1",
+                        "Hymenoptera-1.5Kv1",
+                        "Hymenoptera-2.5Kv2",
+                        "Anthozoa-1.7Kv1"
+                    ]
+                },
+                "target_capture_mode": {
+                    "type": "string",
+                    "default": "illumina",
+                    "description": "Simulate 'illumina' or 'pacbio' reads.",
+                    "enum": ["illumina", "pacbio"]
+                },
+                "target_capture_fmedian": {
+                    "type": "integer",
+                    "default": 2000,
+                    "description": "Median of fragment size at shearing."
+                },
+                "target_capture_fshape": {
+                    "type": "number",
+                    "default": 6.0,
+                    "description": "Shape parameter of the fragment size distribution."
+                },
+                "target_capture_smedian": {
+                    "type": "integer",
+                    "default": 1300,
+                    "description": "Median of fragment size distribution."
+                },
+                "target_capture_sshape": {
+                    "type": "number",
+                    "default": 6.0,
+                    "description": "Shape parameter of the fragment size distribution."
+                },
+                "target_capture_tmedian": {
+                    "type": "integer",
+                    "description": "Median of target fragment size (the fragment size of the data). If specified, will override '--fmedian' and '--smedian'. Othersise will be estimated."
+                },
+                "target_capture_tshape": {
+                    "type": "number",
+                    "description": "Shape parameter of the effective fragment size distribution."
+                },
+                "target_capture_num": {
+                    "type": "integer",
+                    "default": 1000000,
```

```suggestion
                    "default": 500000,
```
according to the nextflow.config

---

Very good catch. I shouldn't have missed that. I currently don't have permission to commit suggestions. @FriederikeHanssen would you be able to commit all the suggestions?